### ISSUE TITLE
DeprecationWarning: distutils re Py3.10

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-core#2696]
Post our recent move to using Python3.9 (#2692) we have the following while runing our tests using rock-tests runner in PyCharm:
```
/opt/rockstor/.venv/lib/python3.9/site-packages/django/utils/version.py:98: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
  loose_version = LooseVersion(version)
```
This looks to be a change coming with Python 3.10 re: **PEP 632 – Deprecate distutils module**
https://peps.python.org/pep-0632/

**Comment by phillxnet** [comment]
The file in question here is form our Django install, we we may just have this resolved once we update our Django:

> /opt/rockstor/.venv/lib/python3.9/site-packages/django/utils/version.py:98

See Milestone: Django 3.2 https://github.com/rockstor/rockstor-core/milestone/24

Which is pending give it is a dependency of:
Py3.10 Milestone: https://github.com/rockstor/rockstor-core/milestone/25

**Comment by phillxnet** [comment]
Re:

> The file in question here is from our Django install, we may just have this resolved once we update our Django:

As of PR: #2734 merge we no longer see this issue.

Closing as:
Fixed by #2737
